composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": ">=5.6",
```

Comment:
Could you please change this to ^5.6 || ^7.0 current expession would also allow php v8. This doesn't exist but it will in the future.

Reply:
Fixed it!